fiftyone/server/routes/camera.py

```
@@ -7,19 +7,255 @@
 """
 
 import logging
-from typing import List, Optional
+from typing import Dict, List, Optional, Set
```

Ruff (0.15.6)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)